docs/development.md

```diff
@@ -67,6 +67,43 @@ You can use the `DOCKER_OPTS` variable to add additional arguments to the Docker
 $ make e2e-docker DOCKER_OPTS=""
 ```
 
+### Local Nomad
+
+In order to support the development of Poseidon, a local Nomad dev server is recommended. Following the instructions below, you can setup a Nomad server on your local system that won't persist any data between restarts.
+
+## macOS
+
+```shell
+brew tap hashicorp/tap
+brew install hashicorp/tap/nomad
+brew services start nomad
+```
+
+**Prerequisites**: [Docker for Mac](https://docs.docker.com/desktop/mac/install/) is installed and started:
+```shell
+brew install --cask docker
+```
+
+**Note**: Due to architecture of Docker networking on macOS, the bridge network is not available with Nomad. Please refer to the [Nomad FAQ](https://www.nomadproject.io/docs/faq#q-how-to-connect-to-my-host-network-when-using-docker-desktop-windows-and-macos) for more information. As a result, those environments having network access enabled won't sync properly to Nomad and thus cannot be started. 
+
+## Linux
+
+```shell
+curl -fsSL https://apt.releases.hashicorp.com/gpg | sudo apt-key add -
+sudo apt-add-repository "deb [arch=amd64] https://apt.releases.hashicorp.com $(lsb_release -cs) main"
+sudo apt-get update && sudo apt-get install nomad
```

Comment:
In the macOS example we also start the server...
```suggestion
sudo apt-get update && sudo apt-get install nomad
sudo nomad agent -dev
```
I slightly prefer this command to start Nomad, because you have all the logs directly in the terminal....

Reply:
I added it for Linux and might also switch the way I start it in macOS (maybe with a new PR) depending on my further experiences.